src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -504,7 +504,8 @@ def test_aks_create_and_update_with_http_proxy_config(self, resource_group, reso
             --resource-group={resource_group} \
             --vnet-name={name} \
             --name proxy-subnet \
-            --address-prefix 10.42.3.0/24'
+            --address-prefix 10.42.3.0/24 \
+            --default-outbound false'
```

Comment:
The change fails the test.

Reply:
At present, it would  be okay to set to true if the test is recording mode, but not sure for future.
It will be temporarily changed to true in https://github.com/Azure/azure-cli/pull/29559